secant/Features/BackupFlow/RecoveryPhraseValidation.swift

```diff
@@ -0,0 +1,231 @@
+//
+//  RecoveryPhraseValidation.swift
+//  secant-testnet
+//
+//  Created by Francisco Gindre on 10/29/21.
+//
+
+import Foundation
+import ComposableArchitecture
+import SwiftUI
+
+typealias RecoveryPhraseValidationStore = Store<RecoveryPhraseValidationState, RecoveryPhraseValidationAction>
+typealias RecoveryPhraseValidationViewStore = ViewStore<RecoveryPhraseValidationState, RecoveryPhraseValidationAction>
+
+/// Represents the completion of a group of recovery words by the addition of one word into the given group
+struct ValidationWord: Equatable {
+    var groupIndex: Int
```

Comment:
One thing that has been tough to digest in this PR, is that we seem to bounce back and forth between the terms "Group" and "Chunk".  I'm guessing this started by introducing a "chunking" function that breaks up an array into an array of arrays, which is a standard term for a function like that.  However, at some point in the line of using said chunks that are derived from that function, the "Chunk" name is used with things like chunkSize or even types like Phrase.Chunk.  

I would suggest for readability we try to keep this consistent by keeping the name "Chunk" for just the function that does said "Chunking", and then consistently use the term "group" afterwards.  I think this would help readers of this code understand how all these different pieces fit together better, because you'd have a consistent way of referencing this portion of your state.  i.e. you would end up with `Phrase.Group` and wordGroupSize or groupSize.

Code like this:
`let groupLength = words.count / Self.phraseChunks`
would read better and make more logical sense if it read:
`let groupLength = words.count / Self.phraseGroups` for example.

Reply:
thanks this is a great suggestion :)